### ISSUE TITLE
Start funding for becoming RNode message

### ISSUE THREAD

**Issue by hvredevoort** [CPChain/chain#5]
Hi there, 
This is CPChain team. Note that this is for bug report. If you are requesting a feature, please select *Feature Request* template. 

**Where does the bug occur?**
[e.g. the chain, the wallet, the console or the explorer]
I repeatedly see a "Start funding for becoming RNode" message on my screen I launched cpchain. 
**Describe the bug**
Yesterday I was able to successfully start cpchain for wallet:
https://cpchain.io/explorer/address/1d93a6d7c9169b06f4c5fb8f9307c69192865ce3
I have been a validator but did not create any blocks yet. I had several invalid address errors so I wiped the ./datadir/cpchain directory and started syncing again.
Currently I have started cpchain 0.2.3 again but keep on seeing "Start funding for becoming RNode" 
So I think there is some action required to get 1d93a6d7c9169b06f4c5fb8f9307c69192865ce3
connected again.
**To Reproduce**
Steps to reproduce the behavior, and the commands you use.

./cpchain run --datadir ./datadir --unlock 1d93a6d7c9169b06f4c5fb8f9307c69192865ce3 --rpcaddr 127.0.0.1:8501 --port 30311 --mine --rpcapi personal,eth,cpc,admission,net,web3,db,txpool,miner --linenumber

**Expected behavior**
Please check my address and allow it to get back as a participating node.

**Screenshots**

![image](https://user-images.githubusercontent.com/6986026/57358709-4a693800-7176-11e9-884b-86fe8e1c72ec.png)

** Configuration (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04]
 - cpchain version [e.g. 0.2.3]

Ubuntu 18.04
cpchain version 0.2.3
16GB, 8 cores, 1250GB disk

**Additional context**
If your bug can only be reproduced occasionally, please emphasize it.

**Comment by hvredevoort** [comment]
I'm running on a virtual server in Azure with 16GB memory, 8 cores and 1250GB SSD

**Comment by alex031029** [comment]
This info is due to a modification of our code. These some log were printed since our smart contract is a bit old, although it does not incur any errors, nor it affects your mining process.. 
TL;DR: simply omit it :)
